Tensile/GenerateSummations.py

```diff
@@ -155,44 +179,49 @@ def GenerateSummations(userArgs):
 
         libSolutions = tensileLibrary["solutions"]
 
-        mapper={}
+        libSolutionNames = []
         for s in libSolutions:
-            key=s["name"]
-            value=s["info"]["SolutionNameMin"]
-            mapper[key]=value
+            kenelName=s["name"]
+            libSolutionNames.append(kenelName)
```

Comment:
Spelling (kernel)

Reply:
Fixed

Reply:
The other misspelling got fixed, but I think you missed this one?